deploy/olm-catalog/ember-csi-operator/ember-csi-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: ember-csi-operator
+channels:
+- name: alpha
+currentCSV: ember-csi-operator.v0.0.1
```

Comment:
This was a mistake, which I hadn't updated yet. It should be nested under 'channels':

e.g.
channels:
- name: alpha
  currentCSV: ember-csi-operator.v0.0.1